src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -514,50 +400,23 @@ def open_archive_dialog(self, slack_request: SlackActionRequest, group: Group) -
             callback_id["channel_id"] = slack_request.data["channel"]["id"]
         callback_id = orjson.dumps(callback_id).decode()
 
-        if features.has("organizations:slack-sdk-action-view-open", group.project.organization):
-            modal_payload = self.build_archive_modal_payload(callback_id)
-            slack_client = SlackSdkClient(integration_id=slack_request.integration.id)
-            try:
-                slack_client.views_open(
-                    trigger_id=slack_request.data["trigger_id"],
-                    view=modal_payload,
-                )
-            except SlackApiError:
-                logger.exception(
-                    "slack.action.response-error",
-                    extra={
-                        "organization_id": org.id,
-                        "integration_id": slack_request.integration.id,
-                        "trigger_id": slack_request.data["trigger_id"],
-                        "dialog": "archive",
-                    },
-                )
-        else:
-            modal_payload = self.build_archive_modal_payload_deprecated(callback_id)
-            payload = {
-                "view": orjson.dumps(modal_payload).decode(),
-                "trigger_id": slack_request.data["trigger_id"],
-            }
-
-            slack_client = SlackClient(integration_id=slack_request.integration.id)
-            try:
-                headers = {"content-type": "application/json; charset=utf-8"}
-                slack_client.post(
-                    "/views.open",
-                    data=orjson.dumps(payload).decode(),
-                    headers=headers,
-                )
-            except ApiError as e:
-                logger.exception(
-                    "slack.action.response-error",
-                    extra={
-                        "error": str(e),
-                        "organization_id": org.id,
-                        "integration_id": slack_request.integration.id,
-                        "trigger_id": slack_request.data["trigger_id"],
-                        "dialog": "archive",
-                    },
-                )
+        modal_payload = self.build_archive_modal_payload(callback_id)
+        slack_client = SlackSdkClient(integration_id=slack_request.integration.id)
+        try:
+            slack_client.views_open(
+                trigger_id=slack_request.data["trigger_id"],
+                view=modal_payload,
+            )
```

Comment:
metrics for success and failure?

Reply:
ill make a separate PR for adding metric here & for valid_channel_id

Reply:
see https://github.com/getsentry/sentry/pull/74214